sections/header.liquid

```diff
@@ -71,6 +71,11 @@
 </style>
 
 {%- style -%}
+  .header {
+    padding-top: {{ section.settings.padding_top | times: 0.5 | round: 0 }}px;
+    padding-bottom: {{ section.settings.padding_bottom | times: 0.5 | round: 0 }}px;
+  }
```

Comment:
We're using `times: 0.75` in other sections it seems. Any reasons here to use `0.5` instead ?

Reply:
I would personally keep it as `0.5` here. A sticky header that is taller has a bigger impact than a section with extra padding at the top, so I think it makes sense to go with a smaller value here. As long as we keep using the same multiples (`4px`), I personally think it's better to have a header that is not too tall than be consistent.

But I'll keep this conversation unresolved so UX can share their thoughts and make a decision.